tests/testthat/test-data.R

```diff
@@ -1,20 +1,17 @@
 test_that("Bucharest dataset includes all elements", {
   expect_setequal(names(bucharest_osm),
-                  c("bb", "aoi_network", "aoi_buildings", "boundary",
-                    "river_centerline", "river_surface", "streets",
-                    "railways", "buildings"))
+                  c("boundary", "river_centerline", "river_surface",
```

Comment:
Do tests pass?  `get_osmdata` seem to return other variables, including `bb`, `aoi_network`, etc. - maybe we need to rerun `data-raw/bucharest.R`?